api/cron/takedown_unpublished_models.py

```diff
@@ -18,11 +20,44 @@
 from utils.deployer import ModelDeployer  # # noqa isort:skip
 
 
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--task_code",
+        type=str,
+        default="",
+        help=("Task code to filter by when choosing what models to delete."),
```

Comment:
What do you think about changing this to:

"The script will take down all unpublished model endpoints for every task. Here, you can additionally specify a task where you want to take down all published model endpoints as well."